root/static/scripts/edit/URLCleanup.js

```diff
@@ -2582,6 +2577,25 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'lantis': {
+    match: [
+      new RegExp(
+        '^(https?://)?(www\\.)?lantis\\.jp/release-item2\\.php\\?' +
+        'id=[0-9a-f]{32}$',
+        'i',
+      ),
+      new RegExp(
+        '^(https?://)?(www\\.)?lantis\\.jp/release-item/' +
+        '[A-Z]+-\\d+(\\.html)?$',
+        'i',
+      ),
+    ],
+    restrict: [LINK_TYPES.discographyentry],
+    clean: function (url) {
+      url = url.replace(/^(?:https?:\/\/)?(?:www\.)?lantis\.jp\/release-item\/([A-Z]+-\d+)(?:\.html)?$/, 'https://www.lantis.jp/release-item/$1.html');
```

Comment:
* Any particular reason for normalizing the scheme to `https`? And only the `/release-item/` URLs? There are currently 600+ `http`Lantis URLs for 200+ `https` Lantis URLs. The preexisting test has no `http`.
* Same questions for the subdomain name `www`. There are currently less than 100 Lantis URLs without  `www`. The preexisting test has no `www`.

Not sure these changes are intentional as they are mentioned neither in the ticket nor in the commit messages.

The other change which auto-add the ending `.html` looks good to me. There is a way to do only this change:

```suggestion
      url = url.replace(/^((?:https?:\/\/)?(?:www\.)?lantis\.jp)\/release-item\/([A-Z]+-\d+)(?:\.html)?$/, '$1/release-item/$2.html');
```

Reply:
The reason for "https" and "www" is that on the website of its parent company - [Bandai Namco Arts](https://www.bandainamcoarts.co.jp/#sites), they link Lantis to https://www.lantis.jp/top.html (Although lantis.jp works with both http/https and with or w/o www).